packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -17,9 +17,9 @@ const allOptions: NodeVersion[] = [
 const supportedOptions = allOptions.filter(o => !isDiscontinued(o));
 
 // This version should match Fargate's default in the PATH
-// Today that is Node 8
+// Today that is Node 12
 export const defaultSelection = supportedOptions.find(
-  o => o.major === 8
+  o => o.major === 12
```

Comment:
Why is the node version being changed in this PR?

This should be a separate PR

Reply:
Will undo, this was just so the tests wouldn't print the warning